test/xpu/test_fusion.py

```diff
@@ -122,6 +122,165 @@ def forward(self, x, other):
+                        kernel_size=3,
+                    )
+                    mod = mod.to(memory_format=memory_format).eval()
+                    with torch.no_grad():
+                        x = x.to("xpu")
+                        mod = mod.to("xpu")
+                        ref = mod(x)
+                        attr = pointwise_info.attr
+                        scalars = pointwise_info.scalars
+                        algorithm = pointwise_info.algorithm
+                        fused = torch.ops.mkldnn._convolution_pointwise(
+                            x,
+                            mod.conv.weight,
+                            mod.conv.bias,
+                            mod.conv.padding,
+                            mod.conv.stride,
+                            mod.conv.dilation,
+                            mod.conv.groups,
+                            attr,
+                            scalars,
+                            algorithm,
+                        )
+                    self.assertEqual(ref, fused)
+
+    def test_conv_binary_fusion_ops(self):
+        class M(nn.Module):
+            def __init__(
+                self,
+                binary_fn,
+                dim,
+                in_channels,
+                out_channels,
+                dilation,
+                groups,
+                bias,
+                **kwargs,
+            ):
+                super().__init__()
+                self.conv = CONV_MODULES[dim](
+                    in_channels,
+                    out_channels,
+                    dilation=dilation,
+                    groups=groups,
+                    bias=bias,
+                    **kwargs,
+                )
+                self.binary = binary_fn
+
+            def forward(self, x, other):
+                x = self.conv(x)
+                x = self.binary(x, other)
+                return x
+
+        for pointwise_name, pointwise_fn in self._binary_list().items():
+            x = torch.randn(
+                (
+                    1,
+                    3,
+                    112,
+                    112,
+                )
+            ).to("xpu")
+            mod = M(pointwise_fn, 2, 3, 3, 1, 1, True, kernel_size=3).to("xpu")
+            other = torch.randn_like(mod.conv(x))
+            with torch.no_grad():
+                ref = mod(x, other)
+                unary_attr = None
+                attr = pointwise_name
+                fused = torch.ops.mkldnn._convolution_pointwise(
+                    x,
+                    other,
+                    mod.conv.weight,
+                    mod.conv.bias,
+                    mod.conv.padding,
+                    mod.conv.stride,
+                    mod.conv.dilation,
+                    mod.conv.groups,
+                    attr,
+                    None,
+                    unary_attr,
+                    [],
+                    None,
+                )
+                if attr == "add":
```

Comment:
why here is only check `add`?

Reply:
only binary add has inplace verion at onednn side.